fluent-plugin-systemd.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: fluent-plugin-systemd
+  version: 1.0.5
+  epoch: 0
+  description: This is a fluentd input plugin. It reads logs from the systemd journal.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ruby-3.2
+      - ruby3.2-fluentd
```

Comment:
I see we're adding the fluentd gem to a few packages at runtime. Are we should it's needed by these individual gems?

Reply:
Yes